integration/linux/run-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "$0")/../.."
+
+tag_ubuntu_1404=dtbosh-ubuntu14.04
+tag_ubuntu_1604=dtbosh-ubuntu16.04
+project=dtbosh_linux
+docker_compose_opts="--project-name $project --file integration/linux/docker-compose.yml"
+
+echo Starting up services...
+if ! docker-compose $docker_compose_opts up --detach --build ; then
```

Comment:
`$docker_compose_opts` isn't quoted.

Reply:
That's on purpose, `$docker_compose_opt` has multiple args appended to the command.

Reply:
Understood :-)